core/primitives/src/version.rs

```diff
@@ -144,7 +147,7 @@ pub const PEER_MIN_ALLOWED_PROTOCOL_VERSION: ProtocolVersion = MAIN_NET_PROTOCOL
 /// Current protocol version used on the main net.
 /// Some features (e. g. FixStorageUsage) require that there is at least one epoch with exactly
 /// the corresponding version
-const MAIN_NET_PROTOCOL_VERSION: ProtocolVersion = 50;
+const MAIN_NET_PROTOCOL_VERSION: ProtocolVersion = 51;
```

Comment:
:thinking: I am a bit surprised that we do version bump here. Was this indeed the case that we upgraded mainnet version? That is, that the next stable release should generally get protocol version 52?

Reply:
It should get version 51 and we already have stabilized feature, that's why I bumped it (for that feature)

Reply:
I'm still confused why we need to bump the version. The protocol version of testnet is already 50 which included the hotfix. So it seems we are bumping protocol version for nothing?